tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = flake8,pydocstyle,docs
+
+
+[testenv]
+basepython=python2.7
+# Installing with sdist makes tox/pip very unhappy.
+usedevelop=True
+# Bodhi requires koji, which isn't available on PyPI.
+sitepackages=True
+
+
+[testenv:docs]
+changedir = docs
+deps =
+    sphinx
+    cornice_sphinx
+whitelist_externals =
+    mkdir
+    rm
+    sphinx-build
+commands=
+    rm -rf _build images
+    mkdir -p _static images
+    ./generate_db_schema
+    sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
+    sphinx-build -W -b man -d {envtmpdir}/doctrees .  _build/man
```

Comment:
I'm also assuming that it'll be possible to view the docs in Jenkies. It might not actually render though, in which case it might not be that valuable. I was just thinking that it could be cool to see what a docs PR would look like without having to check out and build it myself. Pie in the sky.